database/mysql/mysql_user.py

```diff
@@ -341,7 +341,7 @@ def privileges_unpack(priv):
 
     # if we are only specifying something like REQUIRESSL in *.* we still need
     # to add USAGE as a privilege to avoid syntax errors
-    if priv.find('REQUIRESSL') != -1 and 'USAGE' not in output['*.*']:
+    if priv.find('REQUIRESSL') != -1 and not filter(lambda x: x not in [ 'REQUIRESSL' ], output['*.*']):
```

Comment:
@willthames Thanks for your time

	not filter(lambda x: x not in [ 'REQUIRESSL' ], output['*.*']

The meaning of this statement is: output['*.*'] contains other elements than REQUIRESSL


You don' t want to add USAGE when output['*.*'] already contains for example ALL.